packages/google_maps_flutter/google_maps_flutter/README.md

```diff
@@ -46,6 +46,19 @@ This means that app will only be available for users that run Android SDK 20 or
                android:value="YOUR KEY HERE"/>
 ```
 
+#### Hybrid Composition
+
+To use Hybrid Composition to render the `GoogleMap` widget on Android. Set the
```

Comment:
Nit: maybe link "Hybrid Composition" to an explanation of what that is for people who don't know?